Documentation/ceph-upgrade.md

```diff
@@ -204,45 +202,36 @@ time without compatibility support and without prior notice.
 
 Let's get started!
 
-### 1. Configure manifests
```

Comment:
@BlaineEXE Is the "Configure manifests" still applicable for v1.1? I didn't see why they would need to replace the namespaces for all of the manifests. The changes here restrict the yaml updates to only the files related to upgrade.